engines/director/lingo/xlibs/popupmenuxobj.h

```diff
@@ -22,9 +22,17 @@
 #ifndef DIRECTOR_LINGO_XLIBS_POPUPMENUXOBJ_H
 #define DIRECTOR_LINGO_XLIBS_POPUPMENUXOBJ_H
 
+#include "graphics/macgui/macpopupmenu.h"
```

Comment:
Since you forward-declare the `MacPopUp` class below, I recommend not including this file here to reduce header dependency.